scripts/host-display

```diff
@@ -8,108 +8,96 @@ n.b. the value returned by the "status" command will be correct as of the next
-        next boot.
+    Configure the host not to output its console to the physical display on
+    next boot.
     """
     if bootloader.default == XE_SERIAL:
-        doexec([XEN_CMDLINE, "--set-xen", "console=%s" % (_get_com_port())])
+        runCmd([XEN_CMDLINE, "--set-xen", "console=%s" % (_get_com_port())])
     else:
-        doexec([XEN_CMDLINE, "--delete-dom0", "console=tty0"])
-        doexec([XEN_CMDLINE, "--delete-xen", "console"])
+        runCmd([XEN_CMDLINE, "--delete-dom0", "console=tty0"])
+        runCmd([XEN_CMDLINE, "--delete-xen", "console"])
+
 
 def enable(bootloader):
     """
-        Configure the host to output its console to the physical display on
-        next boot.
+    Configure the host to output its console to the physical display on
+    next boot.
     """
     if bootloader.default == XE_SERIAL:
-        doexec([XEN_CMDLINE, "--set-xen", "console=%s,vga" % (_get_com_port())])
+        runCmd([XEN_CMDLINE, "--set-xen", "console=%s,vga" % (_get_com_port())])
     else:
-        doexec([XEN_CMDLINE, "--set-xen", "console=vga"])
-        doexec([XEN_CMDLINE, "--set-dom0", "console=hvc0 console=tty0"])
+        runCmd([XEN_CMDLINE, "--set-xen", "console=vga"])
+        runCmd([XEN_CMDLINE, "--set-dom0", "console=hvc0 console=tty0"])
+
 
 def status(_):
     """
-        Determine from the bootloader config whether the host is currently
-        configured to output its console to the physical display.
+    Determine from the bootloader config whether the host is currently
+    configured to output its console to the physical display.
     """
-    (_, stdout, _) = doexec([XEN_CMDLINE, "--get-xen", "console"])
-    if "vga" in stdout.readline().strip():
+    (_, stdout) = runCmd([XEN_CMDLINE, "--get-xen", "console"], with_stdout=True)
+    if "vga" in stdout:
         sys.stdout.write("enabled")
     else:
         sys.stdout.write("disabled")
 
-COMMANDS = {
-    'disable': disable,
-    'enable': enable,
-    'status': status
-}
+
+COMMANDS = {"disable": disable, "enable": enable, "status": status}
+
 
 def _get_com_port():
     """
-        Determine from the current bootloader config which COM port is in use
+    Determine from the current bootloader config which COM port is in use
     """
-    (_, stdout, _) = doexec([XEN_CMDLINE, "--get-xen", "console"])
-    m = re.match("console=.*(com\d).*", stdout.readline().strip())
+    (_, stdout) = runCmd([XEN_CMDLINE, "--get-xen", "console"], with_stdout=True)
+    m = re.match(r"console=.*(com\d).*", stdout)
     if m:
         return m.group(1)
     else:
         return "com1"
 
+
 def usage():
-    """ Display a usage string. """
-    command_names = list(COMMANDS.keys())
-    command_names.sort()
+    """Display a usage string."""
+    command_names = sorted(COMMANDS.keys())
     print("Usage:")
-    print("    %s {%s}" % (sys.argv[0], '|'.join(command_names)))
+    print("    {} {{{}}}".format(sys.argv[0], "|".join(command_names)))
```

Comment:
The existing version is shorter and more readable to me:
```suggestion
    print("    %s {%s}" % (sys.argv[0], '|'.join(command_names)))
```
Also, please don't mix actually needed cosmetic changes (like removing spaces aboce) with changes needed Python3, this makes it needlessly hard to review the changes.

Reply:
str.format has more performance, but bad readablity in this case, I will revert this change here.